packages/migration/migrations/openhim/20221124202628-user-audit-channels.cjs

```diff
@@ -172,9 +195,26 @@ exports.up = async (db, client) => {
         password: ''
       })
 
+      await addRouteToChannel(db, 'Event Not Duplicate', {
+        type: 'http',
+        status: 'enabled',
+        forwardAuthHeader: true,
+        name: 'Metrics -> Marked as not duplicate',
+        secured: false,
+        host: 'metrics',
+        port: 1050,
+        path: '',
+        pathTransform: '',
+        primary: false,
+        username: '',
+        password: ''
+      })
+
```

Comment:
@sifulovi We need to write a new migration for this. Amending previous migrations is not ideal